cmd/init-plugin.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/go-flutter-desktop/hover/internal/fileutils"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(createPluginCmd)
+}
+
+var createPluginCmd = &cobra.Command{
+	Use:   "init-plugin",
+	Short: "Initialize a go-flutter plugin in a existing flutter platform plugin",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("requires one argument, the VCS repository path. e.g.: github.com/my-organization/" + getPubSpec().Name + "\n" +
+				"This path will be used by Golang to fetch the plugin, make sure it correspond to the code repository of the plugin!")
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		assertInFlutterProject()
+
+		vcsPath := args[0]
+
+		err := os.Mkdir(buildPath, 0775)
+		if err != nil {
+			if os.IsExist(err) {
+				fmt.Println("hover: A file or directory named `" + buildPath + "` already exists. Cannot continue init-plugin.")
+				os.Exit(1)
+			}
+		}
+
+		if _, ok := getPubSpec().Flutter["plugin"]; !ok {
+			fmt.Println("hover: The directory doesn't appear to contain a plugin package. To create a flutter plugin, use `flutter create --template=plugin`.")
```

Comment:
Users probably still need to run hover after they create the flutter plugin, because they want to have a hover plugin.

Maybe change to: "To create a new plugin, first run `flutter create --template=plugin`, then run `hover init-plugin`."